README.md

```diff
@@ -19,8 +19,34 @@
 
 This is the Pangea JavaScript repository.
 
-[Pangea Node SDK](/packages/pangea-node-sdk)
-[Pangea SDK examples](https://github.com/pangeacyber/pangea-javascript/tree/main/examples)
+## Pangea Javascript Tools
+
+This is a list of tools or useful stuff that you will found out in this repository and their links
+
+- Pangea Node SDK [Click here](/packages/pangea-node-sdk)
+- List of Pangea javascript examples per service [Click here](/examples)
+
+## Repository structure
+
+|- **README.md**: This readme file [Click here](/README.md)  
+|- **examples**: SDK examples folder [Clik here](/examples)  
```

Comment:
Can we just have the hyperlink be the SDK examples folder? I think that'd be cleaner

Reply:
We were talking yesterday and we arrive to the conclusion to make it more explicit that user can click and they'll go redirected